### ISSUE TITLE
Machine learning random seed

### ISSUE THREAD

**Issue by EvanYathon** [UBC-MDS/Capital_Bikeshare_Analysis#28]
When I was testing `run_all.sh`, it seemed that there was still an element of randomness somewhere in the decision tree or cross validation.  Sometimes when I displayed the feature importances image the top three would be shuffled - they were always the same top three but the order was inconsistent.

I was thinking we should add a constant state for reproducibility - what are your thoughts?

**Comment by Mikeymice** [comment]
good point. i think we can also use random_state when declaring Decision Tree. I will check it and let you know
